sdk/package.json

```diff
@@ -7,19 +7,19 @@
     "packageManager": "pnpm@8.12.1",
     "workspaces": [
         "core",
-        "client"
+        "msp-client"
```

Comment:
This is not relevant to this PR, right? It's just a fix on the previous one if I understand correctly